app/models/manageiq/providers/amazon/inventory/targets/event_payload_vm.rb

```diff
@@ -0,0 +1,25 @@
+class ManageIQ::Providers::Amazon::Inventory::Targets::EventPayloadVm < ManageIQ::Providers::Amazon::Inventory::Targets
+  def initialize_inventory_collections
+    instance_ems_ref = target[:full_data]["configurationItem"]["resourceId"]
```

Comment:
so this whole Target class only fills the `ems_ref` and the rest is coming from the db?